README.md

```diff
@@ -28,6 +28,10 @@ To use GOV.UK Frontend with NPM you must:
 
 1. Install the long-term support (LTS) version of [Node.js](https://nodejs.org/en/), which includes NPM.
 2. Create a [package.json file](https://docs.npmjs.com/files/package.json) if you don’t already have one. You can create a default `package.json` file by running `npm init` from the root of your application.
+3. If you want to use the GOV.UK Frontend Nunjucks macros, install Nunjucks - the minimum version required is 3.0.0.
```

Comment:
Is this definitely true?

Reply:
Yes it is, see PR description: 

> Document minimum version of Nunjucks in macros: 3.0.0 (2.5.2 and earlier throw up an error due to whitespacing being handled differently)

Also see this: https://github.com/alphagov/govuk-frontend/pull/615#issuecomment-375262011